### PR TITLE
Added retrieval for MessageAttributes

### PR DESCRIPTION
**Pull request by felixlehmann**
As I prefer to work with the MessageAttributes in addition to the body, I added the retrieval for them.

This changes the way the data is returned.

### CODE REVIEW

**Comment by felixlehmann** [comment]
So far I used the repository for receiving messages only.
But in addition to my commit, sending messages with message attributes should be possible, too.

File: src/Shell/Task/QueueWorkerTask.php [hunk 1]
```diff
@@ -145,7 +145,10 @@ protected function _triggerEvent($name, $data = null)
     protected function _work($name, $job)
     {
         foreach ($job->get('Messages') as $message) {
-            $data = json_decode($message['Body'], true);
+            $data = [
+                'Body' => json_decode($message['Body'], true),
+                'MessageAttributes' => $message['MessageAttributes']
+            ];
             $return = call_user_func($this->callbacks[$name], $data, $message['ReceiptHandle']);
```
Comment (lorenzo) [inline src/Shell/Task/QueueWorkerTask.php]: I would add the attributes as another argument here

Comment (felixlehmann) [reply src/Shell/Task/QueueWorkerTask.php]: Can you please get more into detail, where you would like it to be added?

Comment (lorenzo) [reply src/Shell/Task/QueueWorkerTask.php]: yes in `call_user_func`

**Comment by felixlehmann** [comment]
I made the attributes an argument for `call_user_func`

**Comment by lorenzo** [comment]
Thanks!

**Pull request merged**